IKOR-GmbH/sip-framework
# - Fixing mock results not available during SIP test verification

# Description

Bug with test execution: mock activity report is created asynchronously after test is completed.
It happens if middle component is not expected to return response to the caller. Example is route with file consumer.

## Type of change

Please delete options that are not relevant.

# How Has This Been Tested?

- Create a route with file consumer and any producer
- Place middle component (as usual) between consumer and producer
- Create at least two test case scenarios in test-case-definition.yml file. Each test should validate the producer mock
- Execute SIP batch test: first test is successful, all of the following fail with error:
` Error occurred during workflow of the test: Cannot invoke "org.apache.camel.Exchange.getMessage()" because "actualResult" is null`

* SIP Framework version(s): 2.2.1

# Checklist:
